### ISSUE TITLE
Nix shell error: flake does not provide attribute defaultPackage

### ISSUE THREAD

**Issue by tad-lispy** [oxalica/rust-overlay#101]
Hey 👋 Thanks for your work on bringing Rust and Nix together.

While reading the readme I got excited about bringing the whole toolchain in a temporary shell. Unfortunately I can't get it to work:

``` shellsession
$ nix shell github:oxalica/rust-overlay
error: flake 'github:oxalica/rust-overlay' does not provide attribute 'packages.x86_64-linux.defaultPackage.x86_64-linux', 'legacyPackages.x86_64-linux.defaultPackage.x86_64-linux' or 'defaultPackage.x86_64-linux'
```

My `nix-info`:

 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.126, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210317_8a5203d`
 - channels(root): `"nixos-21.11.337975.eabc3821918"`
 - channels(tad): `"home-manager, nixpkgs"`
 - nixpkgs: `/home/tad/.nix-defexpr/channels/nixpkgs`

**Comment by oxalica** [comment]
Fixed in dae73e19607206936d2d50af5332fee5f857cc1c

**Comment by tad-lispy** [comment]
Lovely. Thank you!
